distribution/age10/marketsEmission.json

```diff
@@ -51,52 +51,52 @@
     ],
     "snapshotBlock": 18976728,
     "initialTimestamp": 1704895439,
-    "finalTimestamp": 1710165839,
-    "duration": 5270400
+    "finalTimestamp": 1710082679,
+    "duration": 5187240
   },
   "markets": {
     "0x028171bca77440897b824ca71d1c56cac55b68a3": {
-      "morphoEmittedSupplySide": "6042.406508954061972713",
+      "morphoEmittedSupplySide": "5947.065258710319601422",
```

Comment:
What's the tollerance for these values?

Reply:
my computations are different on the last 7 digits

Reply:
wdym by "tolerance"? It's informational only, its not used in the distribution